dashboard/src2/components/SiteOverview.vue

```diff
@@ -259,27 +270,34 @@ import { renderDialog } from '../utils/components';
 import SiteDailyUsage from './SiteDailyUsage.vue';
 import AlertBanner from './AlertBanner.vue';
 import { trialDays } from '../utils/site';
+import AlertAddPaymentMode from './AlertAddPaymentMode.vue';
 
 export default {
 	name: 'SiteOverview',
 	props: ['site'],
```

Comment:
this file has formatting changes as well, i usually keep formatting commits separate from actual code changes

Reply:
We were fixing the formatting whenever a file is touched, which works for small PRs, but clearly it backfired here 😅.